graphgrid_sdk/ggcore/sdk_messages.py

```diff
@@ -314,3 +314,16 @@ class TrainRequestBody:
     def to_json(self):
         """Encode TrainRequestBody to a json object"""
         return json.dumps(self.__dict__, indent=4)
+
+
+class GetActiveModelResponse(SdkServiceResponse):
+    """Define class representing a get active model api call response."""
+    task: str
+
+    def __init__(self, generic_response: GenericResponse):
+        super().__init__(generic_response)
+
+        if self.status_code == 200:
+            loaded: dict = json.loads(generic_response.response)
+            self.model_name = loaded.get('modelName')
```

Comment:
@NMPowell99  we're trying to keep the java and python responses aligned, could you change this to `self.modelName` and `self.trainedModelData` instead.